libbeat/common/config.go

```diff
@@ -190,6 +202,39 @@ func (c *Config) SetChild(name string, idx int, value *Config) error {
 	return c.access().SetChild(name, idx, value.access(), configOpts...)
 }
 
+func (c *Config) IsDict() bool {
+	return c.access().IsDict()
+}
+
+func (c *Config) IsArray() bool {
+	return c.access().IsArray()
+}
+
+func (c *Config) DebugString() string {
```

Comment:
We can still decide it later. It just looks like something other go-ucfg user could also have a need for.